docs/content/tutorial/step_15.ngdoc

```diff
@@ -0,0 +1,327 @@
+@ngdoc tutorial
+@name 15 - Accessibility
+@step 15
+@description
+
+<ul doc-tutorial-nav="15"></ul>
+
+In this step, we will learn how to fix the accessibility issues in our application.
+
+* We will use the {@link ngAria ngAria} module to enable WAI-ARIA.
+* We will fix some issues in the templates.
+* We will use the [protractor-accessibility-plugin](protractor-a11y-plugin) to review the accessibility of our application when we run our e2e tests.
+
+<div doc-tutorial-reset="15"></div>
+
+## Why accessibility matters
+
```

Comment:
Could you add a paragraph on what is "accessibility" and "aria" and how it affects people. Just a brief summary and then list the links for more info.